Small fixes for the tests

SurduCosmin opened the pull request:
Added missing build folder.
Renamed testConfigExample to testConfig.
Removed wallet_1 function because it was draining the wallet funds.

Commit:
Added build folder to prevent tests from failing

Commit:
Renamed testConfigExample to testConfig to fix test issue

Commit:
Removed function that was draining the test wallet funds

palmerss commented on test/stateless_escrow_timed_lock_test.py:
```diff
@@ -28,7 +28,7 @@ def client(test_config):
 
 
-@pytest.fixture(scope='class')
+#@pytest.fixture(scope='class')
```
this wallet needs to persists across all the tests in this file

palmerss replied:
Seems like we aren't using wallet_2 at all ( I believe when I was originally figuring out this code I had intended for tests that involve wallet_2)
Let's just remove the wallet_2 function all together and uncomment the use of wallet_1 so that the test wallet can receive funding and persist

I think I mentioned to you "lets delete line 31", I meant that we should take out line 31 in the promgram.py file, being that we cant use global.timestamp in a stateless contract..

For some background, this stateless timed escrow was supposed to be an alternate of the stateful timed escrow, however upon learning that we can't use timestamp in stateless escrow we kind of didn't continue with this alternate version, but decided to keep it in a repo as an example of a stateless contract, since they work a little differently.

Commit:
Removed wallet_2
Uncommented usage for the wallet_1
Removed the timestamp for the stateless_escrow as it's impossible to use it
